include/sys/socket.h

```diff
@@ -387,6 +387,37 @@ int socket(int domain, int type, int protocol);
 */
 int getsockname(int socket, struct sockaddr *name, socklen_t *name_len);
 
+/**
+    \brief  Get the name of the connected peer socket.
+
+    This function retrieves the address of the peer connected to the socket
+    specified by the socket descriptor. The address is returned in the buffer
+    pointed to by the addr parameter, and the actual length of the address is
+    returned in the addrlen parameter.
```

Comment:
There some fixing to be done with the coherence of the param names and the documented ones. `name_len` not `address_len` etc.